app/Models/WaitingRoom.php

```diff
@@ -30,6 +30,24 @@ public function users()
     }
 
     /**
+     * Gets a CSV export of all the users in this waiting room.
+     *
+     * @return \League\Csv\ $csv
+     */
+    public function getCSVExport()
```

Comment:
These three `getCSVExport()` functions seem repetitive to me. Is there a way to make this a helper function in `helpers.php` so anything can access it?